### PR TITLE
Update expected output style and browser-specific output

### PR DESCRIPTION
**Pull request by pnevares**
Hey there! 👋 

I ran some examples recently that looked slightly off and updated them here. I also did a review of browser-specific results and updated them based on Chrome 83, Firefox 78, and Safari 13.

Would appreciate any feedback if something was missed. Thank you!

### CODE REVIEW

File: live-examples/js-examples/array/array-foreach.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const array1 = ['a', 'b', 'c'];
 
 array1.forEach(element => console.log(element));
 
-// expected output: "a"
-// expected output: "b"
-// expected output: "c"
+// expected output:
+// "a"
```
Comment (ikarasz) [inline live-examples/js-examples/array/array-foreach.js:6]: Hi @pnevares!
Currently, I am working on unit tests. They will test the real output against the expected ones. Unfortunately, I couldn't find a better solution than adding the "expected output" prefix to each line. So I think we should leave them now as they are.

Comment (pnevares) [reply live-examples/js-examples/array/array-foreach.js:6]: Thanks for the info! I can restore the prefixes in this PR shortly.

**Comment by wbamberg** [comment]
@pnevares , thanks for this, and sorry to be slow reviewing! All these changes look really good to me with the exception of the ones highlighted by @ikarasz . Since the exact formatting of "expected output" is important for unit tests, I'm happy to defer to @ikarasz to give his OK on this pull request.

Thanks for your contribution!

File: live-examples/js-examples/array/array-foreach.js (hunk 1 above)
Comment (pnevares) [reply live-examples/js-examples/array/array-foreach.js]: This part of the PR should be reverted!

**Comment by pnevares** [comment]
Thanks for reviewing! Apologies for adding the "expected output" change to the PR without checking first.

**Pull request merged**